### DISCUSSION TITLE
Issue with ASP.NET Razor Page

### DISCUSSION THREAD

**Discussion by Bishoy-Saber** [GrapesJS/grapesjs#4887] [Q&A]
Hi, I need help in some problems I faced,

I faced a problem while using storagemanger ,I created an API and in urlStore passed this API and nothing happens I wrote autosave:true .
I used editor.getProjectData(), and passed the return value through ajax in OnPost function in razor page ASP.NET and every time the value received by the function is null, while when I used getHTML and getCss and do the same the value is send correctly.
so the problem is How to save canvas data (projectData) or Component with assets and styles and pages it in MS SQL Server and how to view them again in canvas so I can edit them again.

HINT:

I have tried so many code and ways to make it work but I found myself posting this question.
Thanks in advance.

**Comment by artf** [comment]
Duplicate of https://github.com/GrapesJS/grapesjs/discussions/4883
